Technomancy/app.ts

```diff
@@ -2877,34 +2881,16 @@ function save_to_clip() { /* Put save data in clipboard. Copied from Stack Overf
 
     save();
     let text = b64EncodeUnicode(JSON.stringify(localStorage));
-    let textArea: any = document.createElement("textarea");
-
-    /* Styling to make sure it doesn't do much if the element gets rendered */
+    let a: any = document.createElement("a");
 
-    /* Place in top-left corner of screen regardless of scroll position. */
-    textArea.style.position = 'fixed'; textArea.style.top = 0; textArea.style.left = 0; textArea.style.width = '2em'; textArea.style.height = '2em';
+    textArea.href = 'data:text/plain;charset=utf-8,' + encodeURIComponent(text);
+    textArea.download = "Technomancy.sav";
```

Comment:
You're using `a` instead of `textArea`, remember.

Reply:
Gah, my bad. problem with me making changes in the .js, then just replicating in the .ts
Fixed.